Homebrew/emacs_formula.rb

```diff
@@ -81,7 +81,7 @@ def ert_run_tests(*files)
 
   ### Slightly better tested than ert_run_tests
   def byte_compile(*files)
-    emacs_args = %W[ --batch -Q ]
```

Comment:
I removed it because I wasn't able to compile ledger-mode with it—not sure what the problem was.

Moreover since the purpose of `-Q` seems to be to cut down additional steps like loading configuration files, I don't think we're losing anything because we're running in batch mode to begin with anyway. Again, I'm not really a lisp guy, so I don't know if this is right. (Looks right, though.)